ocs_ci/ocs/constants.py

```diff
@@ -730,6 +730,9 @@
 # terraform provider constants
 TERRAFORM_IGNITION_PROVIDER_VERSION = "v2.1.0"
 
+# Minimum storage needed for vSphere Datastore in bytes
+MIN_STORAGE_FOR_DATASTORE = 1.1 * 1024 ** 4
```

Comment:
how about calculating this dynamically, based on the needed OSD size for the deployment and the OS and mon disk size?
Also, we can take into account cluster expansion, to leave some buffer for more OSDs

Reply:
Since OSD and mon disk are type thin provision, there might be discrepancy for MIN_STORAGE_FOR_DATASTORE.
but it will be possible for calculating dynamically. I will take that up in another PR